docs/source/1.0/guides/converting-to-openapi.rst

```diff
@@ -518,6 +518,34 @@ useIntegerType (``boolean``)
             }
         }
 
+.. _generate-openapi-setting-onErrorStatusConflict:
+
+onErrorStatusConflict (``String``)
+    Specifies how to resolve multiple error responses that share the same HTTP status code.
+    This behavior can be customized using the following values for the ``onErrorStatusConflict`` setting:
+
+    * USE_ONEOF
+    Use OpenAPI's ``oneOf`` keyword to combine schemas of errors with same HTTP status code.
```

Comment:
Are these rendering correctly as bulleted lists? I always thought the contents had to be indented like this:
```
* USE_ONEOF
   Use OpenAPI's ....
* USE_PROPERTIES
   blah blah
```

Using a definition list might work better here though:

```
USE_ONEOF
    Use OpenAPI's ``oneOf`` keyword to combine schemas of errors with same HTTP status code.
    ...
USE_PROPERTIES
    .....
```

Also, for naming, in the `unionStrategy` option, we used "oneOf", "object", and "structure". Can we use "oneOf" and "properties" here instead of "USE_ONEOF" and "USE_PROPERTIES"?

Reply:
- Yeah, the bulleted list does render correctly when I tested back then as well as now, using `make serve` command in docs directory. However, definition list was used to replace it anyways, as it does look better. 
- Naming in doc modified to resemble `unionStrategy` section as per the comment & `ErrorStatusConflictHandlingStrategy` enum updated in code to reflect this change.